docs/src/content/docs/commands/BITCOUNT.md

```diff
@@ -7,115 +7,130 @@ The `BITCOUNT` command in DiceDB is used to count the number of set bits (i.e.,
 
 ## Syntax
 
-```plaintext
-BITCOUNT key [start end]
 ```
+BITCOUNT key [start end [BYTE | BIT]]
```

Comment:
Please add a small description for the command.

Reply:
I think it's already added at line no. 6. Do you want me to reshuffle the order or duplicate the description?
Just wanted to confirm, thanks!